bucket/vesta.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+    "version": "3.5.7",
+    "description": "VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies.",
+    "homepage": "https://jp-minerals.org/en/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://jp-minerals.org/vesta/en/download.html"
+    },
+    "url": "https://jp-minerals.org/vesta/archives/3.5.7/VESTA-win64.zip",
+    "architecture": {
+        "64bit": {
+            "url": "https://jp-minerals.org/vesta/archives/3.5.7/VESTA-win64.zip",
+            "hash": "7488ace5ae3a960582e64717a5fbe642f6b4a1e8a715fbd9985f2b836a5e1760",
+            "extract_dir": "VESTA-win64"
+        },
+        "32bit": {
+            "url": "https://jp-minerals.org/vesta/archives/3.5.7/VESTA.zip",
+            "hash": "35c0e7483781398163a5e1472898e4243426691977ca6aee7b4096a41fc10024",
+            "extract_dir": "VESTA"
+        }
+    },
+    "checkver": {
+        "url": "https://jp-minerals.org/vesta/en/download.html",
+        "re": "ver. ([\\d.]+),"
+    },
+    "bin": "VESTA.exe",
```

Comment:
What are available command line parameters?

Reply:
Seems command is useless, drop it.